boot/makebootable.go

```diff
@@ -326,6 +326,7 @@ type makeRunnableOptions struct {
 	Standalone     bool
 	AfterDataReset bool
 	SeedDir        string
+	Unlocker       Unlocker
```

Comment:
nitpick: I think there are too many "Unlocker" things (type, variable, state method), maybe it would help readability to change the variable name here to something else (maybe `StateUnlock` or something like that?)